keel/api/v1/auth/views.py

```diff
@@ -442,10 +443,28 @@ def upload(self, request, format='json'):
         user_id = user.id
         files = request.FILES
 
-        # TODO: move validations to DocumentSerializer
-
         doc_type = req_data.get("doc_type")
 
+        try:
+            doc_serializer = DocumentCreateSerializer(data = request.FILES.dict())
+            doc_serializer.is_valid(raise_exception=True)
+        except ValidationError as e:
```

Comment:
handling an already handled exception...this Would raise your custom error instead of the serializer invalid field error.!

Reply:
Updated, using the default response exception handler.